userguide/snippets/ui_login.rst

```diff
@@ -50,10 +50,10 @@ about the active |ctrlr-term| is displayed on this screen. Log in with:
 On the first login, the EULA found in :ref:`Appendix A` is displayed,
 along with a box where the license key for the %brand% array can be
 pasted. Read the EULA and paste in the license key. High Availability
-(HA) systems must have both active and standby |ctrlrs-term| booted
-before the license key for the HA %brand% system can be entered. The key
-is entered on the active |ctrlr-term|. Click :guilabel:`OK` to save the
-license key and access the |web-ui|.
+(HA) systems must have both |ctrlrs-term-both| booted before the license
```

Comment:
"Booted" implies they are starting up, but really they both have to be fully up.
```suggestion
(HA) systems must have both |ctrlrs-term-both| running before the license
```